Fix imports to use torchao public api

jainapurva opened the pull request:
Summary: Fixing imports to use torchao.utils API, instead of a nested call from torchao.quantization API.

Differential Revision: D65843742

jerryzh168 reviewed (approved):
is this the only one?

jainapurva committed:
Fix imports to use torchao public api (#6797)

Summary:

Fixing imports to use torchao.utils API, instead of a nested call from torchao.quantization API.

Reviewed By: HDCharles, jerryzh168

Differential Revision: D65843742
